src/connector/src/parser/debezium/mongo_json_parser.rs

```diff
@@ -33,6 +33,21 @@ pub struct DebeziumMongoJsonParser {
     id_column: SourceColumnDesc,
     payload_column: SourceColumnDesc,
     source_ctx: SourceContextRef,
+    key_builder: AccessBuilderImpl,
+    payload_builder: AccessBuilderImpl,
+}
+
+// key and payload in DEBEZIUM_MONGO format are accessed in different ways
+fn build_accessor_builder(config: EncodingProperties) -> anyhow::Result<AccessBuilderImpl> {
+    match config {
+        EncodingProperties::Json(_) => Ok(AccessBuilderImpl::DebeziumJson(
+            DebeziumJsonAccessBuilder::new()?,
+        )),
```

Comment:
When will we reach this arm? It seems that only `MongoJson` is used when calling `build_accessor_builder`.

Reply:
removed.